pkg/kv/kvclient/kvcoord/dist_sender_mux_rangefeed.go

```diff
@@ -251,10 +251,12 @@ func (m *rangefeedMuxer) startSingleRangeFeed(
 func (s *activeMuxRangeFeed) start(ctx context.Context, m *rangefeedMuxer) error {
 	streamID := atomic.AddInt64(&m.seqID, 1)
 
+	startTime := timeutil.Now()
```

Comment:
Let's use the new crtime.NowMono() for this. It's a little bit more efficient.

```golang
startTime := crtime.NowMono()
// ...
m.metrics.RangefeedCatchUpBlockedNanos.Inc(startTime.Elapsed().Nanoseconds())

Reply:
TIL, done.
<!-- Reviewable comment -ODS669D851L2lMDgMX3:bj9zbl2 -->
<!-- Sent from Reviewable.io -->